timesketch/lib/analyzers/phishy_domains.py

```diff
@@ -19,10 +19,10 @@
 from timesketch.lib.analyzers import manager
 
 
-class DomainsSketchPlugin(interface.BaseSketchAnalyzer):
-    """Sketch analyzer for domains."""
+class PhishyDomainsSketchPlugin(interface.BaseSketchAnalyzer):
+    """Sketch analyzer for phishy domains."""
 
-    NAME = 'domains'
+    NAME = 'phishy_domains'
 
```

Comment:
once #767  is in, please add the DEPENDENCIES to depend on the domain sketch analyzer

Reply:
Done in the latest refactor

Reply:
Done in the latest refactor